buildkite/bazel-central-registry/bcr_postsubmit.py

```diff
@@ -30,10 +30,12 @@
 import requests
 import subprocess
 import sys
+import tempfile
 
 BCR_BUCKET = "gs://bcr.bazel.build/"
 ATTESTATION_METADATA_FILE = "attestations.json"
 FILES_WITH_ATTESTATIONS = ("source.json", "MODULE.bazel")
+LAST_GREEN_FILE = "last_green.txt"
```

Comment:
Can you add a comment to explain what this is? Or maybe rename to `last_synced_commit`?